### PR TITLE
Create the "xml" and "yaml" plugins tests

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Create the "xml" and "yaml" plugins tests

### What issues does this PR fix or reference?

Issue: https://github.com/eclipse/che/issues/19178
Issue: https://github.com/eclipse/che/issues/19177

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by ScrewTSW — APPROVED** [review]
LGTM

File: tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    });
+
+    suite('Check workspace readiness to work', async () => {
+        test('Wait until project is imported', async () => {
+            await projectTree.openProjectTreeContainer();
+            await projectTree.waitProjectImported(projectName, 'app');
+        });
+    });
+
+    suite('Check the "vscode-xml" plugin', async () => {
```
Comment (JPinkney) [inline tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts]:
```suggestion
    suite('Check the "vscode-yaml" plugin', async () => {
```

File: tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            await editor.type(yamlFileName, Key.SPACE, 19);
+            await editor.waitErrorInLine(19);
+        });
+
+        test('Check error disappearance', async () => {
+            await editor.performKeyCombination(yamlFileName, Key.BACK_SPACE);
+            await editor.waitErrorInLineDisappearance(19);
+        });
+
+        test('To unformat the "xml" file', async () => {
```
Comment (JPinkney) [inline tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts]:
```suggestion
        test('To unformat the "yaml" file', async () => {
```

File: tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            const expectedTextBeforeFormating: string = 'uri:     "timer:tick"';
+            await projectTree.expandPathAndOpenFile(pathToFile, yamlFileName);
+
+            await editor.selectTab(yamlFileName);
+            await editor.moveCursorToLineAndChar(yamlFileName, 19, 10);
+            await editor.performKeyCombination(yamlFileName, Key.chord(Key.SPACE, Key.SPACE, Key.SPACE, Key.SPACE));
+            await editor.waitText(yamlFileName, expectedTextBeforeFormating, 60000, 10000);
+        });
+
+        test('To format the "xml" document', async () => {
```
Comment (JPinkney) [inline tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts]:
```suggestion
        test('To format the "yaml" document', async () => {
```

File: tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts (hunk 1 above)
Comment (Ohrimenko1988) [reply tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts]: Done, thank you for your attention.

File: tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts (hunk 2 above)
Comment (Ohrimenko1988) [reply tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts]: https://github.com/eclipse/che/pull/19217#discussion_r588361126

File: tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts (hunk 3 above)
Comment (Ohrimenko1988) [reply tests/e2e/tests/plugins/VscodeYamlPlugin.spec.ts]: https://github.com/eclipse/che/pull/19217#discussion_r588361126

**Comment by Ohrimenko1988** [comment]
[crw-ci-test]

**Pull request merged**